server/api/api.go

```diff
@@ -29,6 +29,9 @@ type Subscriptions interface {
 
 type Calendar interface {
 	ViewCalendar(from, to time.Time) ([]*remote.Event, error)
+	CreateEvent(event *remote.Event) (*remote.Event, error)
+	CreateCalendar(calendarName string) (*remote.Calendar, error)
+	DeleteCalendarByID(ID string) error
```

Comment:
1. 3/5 `DeleteCalendar` is good, the convention is for a `Delete` to take an ID anyways.
2. 3/5 `s/ID string/calendarID string` - I'd like to see all *ID variable names to be spelled out. Ok to have a .ID field though.

Reply:
1) Agree.  I completely agree.  
2) Agree again.

will make both changes.